tools/telemetry-generator/package.json

```
@@ -35,13 +35,20 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
+		"@fluid-internal/mocha-test-setup": "~2.0.5",
```

Latest version. Also update the lockfile after this.
```suggestion
		"@fluid-internal/mocha-test-setup": "^2.3.1",
```